panels/user-accounts/cc-app-permissions.c

```diff
@@ -995,6 +1025,8 @@ cc_app_permissions_class_init (CcAppPermissionsClass *klass)
   gtk_widget_class_bind_template_child (widget_class, CcAppPermissions, allow_web_browsers_switch);
   gtk_widget_class_bind_template_child (widget_class, CcAppPermissions, restriction_button);
   gtk_widget_class_bind_template_child (widget_class, CcAppPermissions, restriction_popover);
+  gtk_widget_class_bind_template_child (widget_class, CcAppPermissions, system_installation_label);
+  gtk_widget_class_bind_template_child (widget_class, CcAppPermissions, user_installation_label);
```

Comment:
Binding (in the .ui file) the visibility of those labels to the widgets they refer to is a better approach.

Reply:
I don't get it. How would I toggle/decide the visibility of those labels as I have done above (in `update_allow_app_installation`) ?

Reply:
You bind the `visibile` property to the corresponding switches' visibility?

Reply:
Here's an example: https://github.com/endlessm/gnome-control-center/blob/master/shell/cc-window.ui#L63

Reply:
Thanks and addressed.